src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractCreatePdpFeedServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractCreatePdpFeedService;
+import edu.cornell.kfs.concur.service.ConcurRequestExtractFileService;
+
+public class ConcurRequestExtractCreatePdpFeedServiceImpl implements ConcurRequestExtractCreatePdpFeedService {
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractCreatePdpFeedServiceImpl.class);
+	protected ConcurRequestExtractFileService concurRequestExtractFileService;
+
+	@Override
+    public void createPdpFeedsFromRequestExtracts() {
+        /* Get list of all Request Extract files that have not been processed into PDP Import files
+         * If there are files to process then
+         *     Loop through list
+         *        call method this.createPdpFeedFromRequestExtract() to process each file
+         * Else
+         *     report-log no files found to process
+         */
+    }
+
+    private void createPdpFeedFromRequestExtract() {
+        /* Validate request extract header row to file contents
```

Comment:
Same as above, per out Clean Code book, we want to avoid comments. You could however create stubs for the methods to be called inside this method. For the validation part you will probably want to call one single validation method that will encapsulate all the validation details.